libdevcore/OverlayDB.cpp

```diff
@@ -50,33 +40,36 @@ void OverlayDB::commit()
 			for (auto const& i: m_main)
 			{
 				if (i.second.second)
-					batch.Put(ldb::Slice((char const*)i.first.data(), i.first.size), ldb::Slice(i.second.first.data(), i.second.first.size()));
+					writeBatch->insert(db::Slice(reinterpret_cast<char const*>(i.first.data()), i.first.size), db::Slice(reinterpret_cast<char const*>(i.second.first.data()), i.second.first.size()));
```

Comment:
Too complicated expression, could you please make a helper function for converting `h256` to `Slice`?
Second `reinterpet_cast` here is not needed